.gitignore

```diff
@@ -10,6 +10,17 @@ exp*.py
 venv/
 act/
 .actrc
+.Python
+[Bb]in
+[Ii]nclude
+[Ll]ib
+[Ll]ib64
+[Ll]ocal
+[Ss]cripts
```

Comment:
Should scripts be in here? That's a directory used today to store scripts in

Reply:
That's interesting actually - on Windows, if you create a virtual environment, it generates a `Scripts` directory, where the `activate` script is stored. I can add a couple of exceptions to ignore the scripts already in there if that suits?

Reply:
That's an interesting to know! In other words having the `scripts` folder as is can cause issues for Windows? So in other words that directory should maybe be moved?

Reply:
Exactly. Either moved or renamed I reckon, something like `ff1_scripts` could do well

Reply:
That is a good catch and should have it's own issue I think

Reply:
That's absolutely fine. I'll update this PR to not have the `Scripts` directory in the gitignore and open an issue for the other one

Reply:
Sure thing, I've removed it just now but I've also created an issue for that: https://github.com/theOehrly/Fast-F1/issues/265

Reply:
I have to fully disagree with this PR. You should not create a venv inside the project root directory. You should use a separate folder for this. See my explanation in #261

Reply:
Ahh fair enough, that makes sense. I'll leave this one.